src/modules/systemlib/param/param_shmem.c

```diff
@@ -677,6 +677,9 @@ param_set_no_notification(param_t param, const void *val)
 bool
 param_used(param_t param)
 {
+	// TODO FIXME: for now all params are used
```

Comment:
My apologies - I did miss the file title while skimming through the diff. Will merge.